### ISSUE TITLE
Remove automatic save

### ISSUE THREAD

**Issue by adler-j** [davlars/ad-skull-reconstruction#1]
Currently the scripts save to disk with a hard-coded path. We should remove this so that we dont get colliding writes to the NAS.

**Comment by davlars** [comment]
Should have been fixed by merging the pull request by @adler-j. Great work!

**Comment by davlars** [comment]
Realized it's still hardcoded... I'll simply recode it to a local repo, awaiting a more clever solution (e.g. giving save path as input)

**Comment by davlars** [comment]
Now changed to a hardcoded local path (not to disc). Code-wise still as bad, but until I fix a savePath user input, it's better than before.

**Comment by adler-j** [comment]
The current path is not guaranteed to exist. I would frankly just remove the saving for now, or at the very least write straight to the data folder.

**Comment by davlars** [comment]
Right. I however don't mind having it there, just to point out the ways of saving it (even though it's somewhat trivial). Witht the ```saveCont = False ``` set as default in all the scripts, at least the user have to think before saving. As you pointed out earlier it's maybe not a good idea having people write straight to the disc (risking that it get's real messy) so let's keep it as this default local folder. I'm anyhow reopening the issue (again).
